### PR TITLE
Add "Next rank" to rank command

### PR DESCRIPTION
**Pull request by Awakened-Redstone**

### CODE REVIEW

File: src/commands/level.py [hunk 1]
```diff
@@ -34,11 +34,18 @@ async def rank(message: discord.Message, parameters: str, client: discord.Client
 
     avatar = member.avatar_url_as(format=None, static_format='png', size=1024)
 
+    ranks = configuration.LEVEL_ROLES.filter(lambda key, value: level - 1 < value[1])
+    if ranks.lenght() > 0:
+        rank = ranks[ranks.lenght() - 1].value()
```
Comment (Bobby-McBobface) [inline src/commands/level.py:39]:
```suggestion
        rank = ranks[len(ranks) - 1].value()
```

File: src/commands/level.py [hunk 2]
```diff
@@ -34,11 +34,18 @@ async def rank(message: discord.Message, parameters: str, client: discord.Client
 
     avatar = member.avatar_url_as(format=None, static_format='png', size=1024)
 
+    ranks = configuration.LEVEL_ROLES.filter(lambda key, value: level - 1 < value[1])
+    if ranks.lenght() > 0:
```
Comment (Bobby-McBobface) [inline src/commands/level.py:38]:
```suggestion
    if len(ranks) > 0:
```

File: src/commands/level.py [hunk 3]
```diff
@@ -34,11 +34,18 @@ async def rank(message: discord.Message, parameters: str, client: discord.Client
 
     avatar = member.avatar_url_as(format=None, static_format='png', size=1024)
 
+    ranks = configuration.LEVEL_ROLES.filter(lambda key, value: level - 1 < value[1])
```
Comment (Bobby-McBobface) [inline src/commands/level.py:37]: Filter not a attribute of dict

**Review by Bobby-McBobface — CHANGES_REQUESTED** [review]
+7 lines, +3 errors. Do you mind testing before submitting a pull reuqest?

File: src/commands/level.py (hunk 3 above)
Comment (Awakened-Redstone) [reply src/commands/level.py:37]:
```suggestion
    ranks = dict(filter(lambda elem: level < elem[1][1], configuration.LEVEL_ROLES.items()))
```

File: src/commands/level.py [hunk 4]
```diff
@@ -34,11 +34,18 @@ async def rank(message: discord.Message, parameters: str, client: discord.Client
 
     avatar = member.avatar_url_as(format=None, static_format='png', size=1024)
 
+    ranks = dict(filter(lambda elem: level < elem[1][1], configuration.LEVEL_ROLES.items()))
```
Comment (Bobby-McBobface) [inline src/commands/level.py:37]:
```suggestion
    ranks = dict(filter(lambda elem: level - 1< elem[1][1], configuration.LEVEL_ROLES.items()))
```
Compensate for #9

**Review by Bobby-McBobface — CHANGES_REQUESTED** [review]

File: src/commands/level.py [hunk 5]
```diff
@@ -34,11 +34,18 @@ async def rank(message: discord.Message, parameters: str, client: discord.Client
 
     avatar = member.avatar_url_as(format=None, static_format='png', size=1024)
 
+    ranks = dict(filter(lambda elem: level - 1< elem[1][1], configuration.LEVEL_ROLES.items()))
+    if len(ranks) > 0:
+        rank = ranks[len(ranks) - 1].value()
```
Comment (Awakened-Redstone) [inline src/commands/level.py:39]:
```suggestion
        rank = list(ranks.items())[len(ranks) - 1][1]
```

File: src/commands/level.py [hunk 6]
```diff
@@ -34,11 +34,18 @@ async def rank(message: discord.Message, parameters: str, client: discord.Client
 
     avatar = member.avatar_url_as(format=None, static_format='png', size=1024)
 
+    ranks = dict(filter(lambda elem: level - 1< elem[1][1], configuration.LEVEL_ROLES.items()))
```
Comment (Awakened-Redstone) [inline src/commands/level.py:37]:
```suggestion
    ranks = dict(filter(lambda elem: user_xp[1] - 1 < elem[1][1], configuration.LEVEL_ROLES.items()))
```

**Review by Bobby-McBobface — APPROVED** [review]

**Pull request merged**